src/WebUI/Types.d.ts

```diff
@@ -72,4 +72,6 @@ export interface IVssComponentProperties extends IBaseProps {
      * Any of the properties MAY be accessed as an IObservable.
      */
     [property: string]: IObservable<any> | any;
+
+    nameFilterKey?:string;

```

Comment:
this may not fit here, in future if we have to filter by something else, then the name may not be ideal. so it is okay if we have this in the individual component

Reply:
Removed it from the common properties and moved to individual elements.